### ISSUE TITLE
Github logo at top right not working for validkube.

### ISSUE THREAD

**Issue by 7h3-3mp7y-m4n** [komodorio/validkube#64]
While moving the cursor toward the GitHub logo at the top right of the website, the logo should redirect to the desired repo , but is not redirecting itself to the Komodor repository as it should.
<img width="1440" alt="Screenshot 2023-10-02 at 8 35 00 AM" src="https://github.com/komodorio/validkube/assets/115151332/226a0226-444e-4b3a-8e6d-fda39a3c16cf">

**Comment by shivaabhishek07** [comment]
It is working when you click the star button which is just beside the GitHub logo. But it should change to get redirected when you click on GitHub logo..

**Comment by 7h3-3mp7y-m4n** [comment]
Yeah it's true , but wouldn't it be nicer if the logo redirects to the GitHub page ? @shivaabhishek07

**Comment by 7h3-3mp7y-m4n** [comment]
That's my point. Can I work on this ? @shivaabhishek07

**Comment by shivaabhishek07** [comment]
Yeah, you can go ahead and fix this issue. But currently, Komodor has a lack of maintainers(I think) since their last PR merged in March. Once try contacting @nirsht

**Comment by 7h3-3mp7y-m4n** [comment]
okay @shivaabhishek07 .

**Comment by 7h3-3mp7y-m4n** [comment]
I reached @nirsht  and I couldn't get  any response . So I'm thinking to raise a pr and left it for review .. what you say @shivaabhishek07 ?

**Comment by shivaabhishek07** [comment]
Yeah @7h3-3mp7y-m4n , it's better to raise a PR. Maintainers will take care of whether including that feature or not..

**Comment by 7h3-3mp7y-m4n** [comment]
@shivaabhishek07  got it .
